pepper-apis/src/test/java/org/broadinstitute/ddp/export/DataExporterTest.java

```diff
@@ -209,7 +211,17 @@ public void testExtractParticipants() {
             long completeTimestamp = Instant.now().toEpochMilli();
             handle.attach(ActivityInstanceStatusDao.class)
                     .insertStatus(instanceDto.getId(), InstanceStatusType.COMPLETE, completeTimestamp, testData.getUserGuid());
-
+            String invitationCode1 = UUID.randomUUID().toString();
+            String email1 = testData.getUserGuid() + "@" + invitationCode1;
+            handle.attach(InvitationFactory.class)
+                    .createInvitation(InvitationType.RECRUITMENT, invitationCode1, testData.getStudyId(),
+                            testData.getUserId(), email1);
+
+            String invitationCode2 = UUID.randomUUID().toString();
+            String email2 = testData.getUserGuid() + "@" + invitationCode2;
+            handle.attach(InvitationFactory.class)
+                    .createInvitation(InvitationType.AGE_UP, invitationCode2, testData.getStudyId(),
+                            testData.getUserId(), email2);
             // Extract and test
```

Comment:
@yufengwng Regarding your concern about the one to many relationship. I was also was worried about which is why I added this chunk of code. I am confident we are handling it correctly.
(This is one of the things I really don't like about JDBI: one to many handling is super klutzy and forces redundant code. But that is for another day)